Modules/+Experiments/+AutoExperiment/@AutoExperiment_invisible/run.m

```diff
@@ -69,21 +75,28 @@ function run(obj,status,managers,ax)
 try
     for repetition = 1:obj.repeat
         for i=1:size(run_queue,1)
-            status.String = 'Searching for next experiment to run...';
             try
-                drawnow limitrate;
-                assert(~obj.abort_request,'User aborted'); % Allows aborting before running any experiments
                 site_index = run_queue(i,1);
                 exp_index = run_queue(i,2);
+                status.String = sprintf('Searching for next experiment to run\nSite: %i/%i\nExperiment: %i/%i)...',site_index,nsites,exp_index,nexps);
```

Comment:
Is this searching process long enough to warrant a searching message? If so, why does it take so long??

Reply:
It must because people have commented that it seems frozen when it likely is not. It takes so long because if you don't have an analysis file supplied, it goes through every patch function which all apply a fit to see if it needs to continue and what it needs to do if so.